helm/chaos-mesh/values.yaml

```diff
@@ -89,3 +95,6 @@ dashboard:
   volume:
     storage: 3Gi
     storageClassName: standard
+
+jobs:
+  podAnnotations: {}
```

Comment:
Please separate `pre-install` and `post-delete`job configuration and add end-of-line in the end.